storm-core/src/clj/backtype/storm/cluster.clj

```diff
@@ -248,9 +256,9 @@
   [id]
   (str ASSIGNMENTS-SUBTREE "/" id))
 
-(defn code-distributor-path
-  [id]
-  (str CODE-DISTRIBUTOR-SUBTREE "/" id))
+(defn blobstore-path
+  [key]
+  (str BLOBSTORE-SUBTREE "/" key))
```

Comment:
This makes me a bit nervous, as key is a user provided value, and if we put in a `..` or a `/` or something else malicious it could cause a security issue.  Are we checking elsewhere to prevent the key from having potentially harmful characters in it?